package.js

```diff
@@ -12,5 +12,5 @@ Package.onUse(function _(api) {
     api.use(['socialize:server-presence@1.0.0', 'socialize:user-model@1.0.0']);
 
     api.mainModule('server/server.js', 'server');
-    api.mainModule('common/common.js');
+    api.mainModule('common/common.js', 'client');
```

Comment:
After looking at the code structure, it looks like anything that is common code is actually already imported into server so while the current code works, it's slightly redundant.  We can rename common to client to reflect this. Nice catch here.